packages/pwa-buildpack/lib/WebpackTools/configureWebpack/getClientConfig.js

```diff
@@ -199,22 +199,24 @@ async function getClientConfig(opts) {
         }
     } else if (mode === 'production') {
         let versionBanner = '';
-        try {
-            versionBanner = projectConfig.section('staging').buildId;
-            if (!versionBanner || versionBanner.trim().length === 0) {
-                throw new Error('invalid build id');
-            }
-        } catch (error) {
-            try {
-                versionBanner = require('child_process')
-                    .execSync('git describe --long --always --dirty=-dev')
-                    .toString();
-            } catch (e) {
-                versionBanner = `${
-                    require(path.resolve(context, './package.json')).version
-                }-[hash]`;
-            }
-        }
+        const packageJson = require(path.resolve(context, './package.json'));
+        const packageRegex = /^@magento|^@apollo/;
+        const pwaStudioVersions = {
+            'pwa-studio': packageJson.version,
+            ...Object.fromEntries(
+                Object.entries(packageJson.dependencies || {}).filter(
+                    ([packageKey]) => packageRegex.test(packageKey)
+                )
+            ),
+            ...Object.fromEntries(
+                Object.entries(packageJson.devDependencies || {}).filter(
+                    ([packageKey]) => packageRegex.test(packageKey)
+                )
+            )
+        };
+        versionBanner = Object.entries(pwaStudioVersions)
```

Comment:
+1 bonus point if we add a `sort()` at the end 😁

Reply:
Done!
Sample:
```
/*! @version pwa-studio: 12.1.0, build-id: testversion, @magento/venia-ui: ~9.1.0, @magento/upward-security-headers: ~1.0.5, @magento/pwa-theme-venia: ~1.1.0, @magento/pwa-buildpack: ~11.0.0, @magento/peregrine: ~12.1.0, @magento/pagebuilder: ~7.0.1, @magento/eslint-config: ~1.5.0, @magento/babel-preset-peregrine: ~1.2.0, @apollo/client: ~3.4.0 */
```